### PR TITLE
[WIP] Add end-to-end testing

### PR DESCRIPTION
**Pull request by mrogers950**
Rough draft right now, currently working out the setup and teardown steps.

### CODE REVIEW

File: pkg/controller/fileintegrity/config_defaults.go [hunk 1]
```diff
@@ -15,10 +19,6 @@ var aideReinitContainerScript = `#!/bin/sh
 `
 
 var aideScript = `#!/bin/sh
-    if test ! -f /hostroot/etc/kubernetes/aide.db.gz; then
-      echo "initializing AIDE db"
-      aide -c /tmp/aide.conf -i
-    fi
     while true; do
```
Comment (JAORMX) [inline pkg/controller/fileintegrity/config_defaults.go:22]: this is a pretty relevant change IMO. Lets pull into a separate PR.

**Review by JAORMX — CHANGES_REQUESTED** [review]
Just a minor comment. The overall structure and idea looks good!

**Comment by mrogers950** [comment]
obsoleted by #25
